orchagent/aclorch.cpp

```diff
@@ -153,6 +153,244 @@ static const acl_capabilities_t defaultAclActionsSupported =
     }
 };
 
+static acl_table_action_list_lookup_t defaultAclActionList = 
+{
+    {
+        // L3
+        TABLE_TYPE_L3,
+        {
+            {
+                ACL_STAGE_INGRESS,
+                AclActionCapabilities
+                {
+                    {
+                        SAI_ACL_ACTION_TYPE_PACKET_ACTION
+                    },
+                    false
```

Comment:
I guess this boolean isn't used. Maybe you could create another type instead of using acl_capabilities_t

Reply:
Good idea. I use the `set<sai_acl_action_type_t>` directly. Thanks